### PR TITLE
[docs] adds new FAQ entries

### PR DESCRIPTION
**Pull request by cjdb**
We've observed the following things cropping up regularly enough to
address as 'frequently asked':

* Why isn't there a Carbon logo?
* Where should I ask questions?
* What are the prerequisites for building Carbon?
* Why aren't people willing to discuss certain topics?

### CODE REVIEW

File: docs/project/faq.md [hunk 1]
```diff
@@ -182,6 +197,62 @@ If a migration occurs, we anticipate:
...
+decisions. It's not that they're _definitely_ right, or that they are truly set
+in stone. It's just a prioritization of making progress on the open questions
+because at least we have an answer and that answer is probably good enough for
+now. The signal to really revisit stuff is new information. There was an example
+with digit separators: we had never considered how many _domain_ conventions we
+would want to support and so had overly restricted where the separators go.
+
+We encourage folks to either shrug and let quirky syntax be quirky, or look for
+the objective use case that was clearly missed in the previous decision and
+makes the current state _sharply_ problematic before pushing for change.
```
Comment (josh11b) [inline docs/project/faq.md]: I feel like we should say something along the lines of:

When there is disagreement between options, comments saying "I like X" or "I don't like X" don't really contribute to the conversation. Much more helpful would be a reason for or against X that hasn't already been considered.

Comment (cjdb) [reply docs/project/faq.md]: Done.

File: docs/project/faq.md [hunk 2]
```diff
@@ -182,6 +197,67 @@ If a migration occurs, we anticipate:
...
+
+Please ask questions and hold discussions either by using
+[GitHub Discussions](https://github.com/carbon-language/carbon-lang/discussions)
+or the
+[Carbon Language Discord server](https://discord.com/channels/655572317891461132/996113200464543804).
+
+GitHub Issues should be reserved for missing features, bugs, and anything else
+that is fixable by way of a Pull Request.
+
+### What are the prerequisites for contributing to Carbon Language's design and tools??
```
Comment (jonmeow) [inline docs/project/faq.md]:
```suggestion
### What are the prerequisites for contributing to Carbon Language's design and tools?
```

File: docs/project/faq.md [hunk 3]
```diff
@@ -182,6 +197,67 @@ If a migration occurs, we anticipate:
     interoperability wrappers.
 -   Test-driven verification that migrations are correct.
 
+### Where should I ask questions about Carbon Language?
```
Comment (jonmeow) [inline docs/project/faq.md]: Would this make sense under "What is Carbon's status?", right after the demo question?

File: docs/project/faq.md [hunk 4]
```diff
@@ -182,6 +197,67 @@ If a migration occurs, we anticipate:
...
+
+Please ask questions and hold discussions either by using
+[GitHub Discussions](https://github.com/carbon-language/carbon-lang/discussions)
+or the
+[Carbon Language Discord server](https://discord.com/channels/655572317891461132/996113200464543804).
```
Comment (jonmeow) [inline docs/project/faq.md]:
```suggestion
or
[#language-questions on Discord](https://discord.com/channels/655572317891461132/998959756045713438).
```

I admit I'm not sure whether we should point at the server versus the server invite, but let's use #language-questions.

File: docs/project/faq.md [hunk 5]
```diff
@@ -182,6 +197,67 @@ If a migration occurs, we anticipate:
...
+order of magnitude growth in the community. We will do this by filing an issue
+for leads with what new information is available, and we will reevaluate the
+choice.
+
+For example, we're doing this with digit separators: we missed important
+_domain_ conventions and had overly restricted where separators are allowed,
+[an issue was filed with the new information](https://github.com/carbon-language/carbon-lang/issues/1485),
+and we're fixing the choice.
+
+When there is disagreement between options, comments saying "I like X" and "I
```
Comment (jonmeow) [inline docs/project/faq.md]:
```suggestion
When there is a discussion about options or disagreement with a decision, comments saying "I like X" and "I
```

WDYT of this? "disagreement between options" is feeling odd to me.

**Comment by jonmeow** [comment]
Thanks for these additions. :)

File: docs/project/faq.md [hunk 6]
```diff
@@ -182,6 +197,67 @@ If a migration occurs, we anticipate:
...
+For example, we're doing this with digit separators: we missed important
+_domain_ conventions and had overly restricted where separators are allowed,
+[an issue was filed with the new information](https://github.com/carbon-language/carbon-lang/issues/1485),
+and we're fixing the choice.
+
+When there is a discussion about options or disagreement with a decision, comments saying "I like X" and "I
+don't like X" do not contribute to the conversation. We encourage folks to
+either shrug and let quirky syntax be quirky, or look for the objective use case
+that was clearly missed in the previous decision and makes the current state
+_sharply_ problematic before pushing for change.
```
Comment (chandlerc) [inline docs/project/faq.md]: Maybe also recommend using reactions to capture sentiment? I'm thinking something like:

```suggestion
When there is a discussion about options or disagreement with a decision, comments saying "I like X" and "I
don't like X" do not contribute to the conversation. You should use the reactions to signal this kind of sentiment, and we'll pay attention to those. Beyond that, we encourage folks to
either shrug and let quirky syntax be quirky, or look for the objective use case
that was clearly missed in the previous decision and makes the current state
_sharply_ problematic before pushing for change.
```

**Review by chandlerc — COMMENTED** [review]
No concerns here, and don't block on me at least. A minor suggestion below.

**Review by jonmeow — APPROVED** [review]

**Comment by jonmeow** [comment]
Thank you for contributing these FAQ entries!

**Pull request merged**